icons/log-16.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="16" height="17" viewBox="0 0 16 17"><path d="M5 9.25a.75.75 0 01.75-.75h4a.75.75 0 010 1.5h-4A.75.75 0 015 9.25zM4 11.5A.75.75 0 004 13h4a.75.75 0 000-1.5H4z"/><path fill-rule="evenodd" d="M13 .995H3a3 3 0 00-3 3c0 .676.224 1.254.603 1.722.526.65 1.331.783 1.907.783h1.177c-.364.662-.814 1.339-1.287 2.048-.205.309-.414.624-.623.946C.891 10.865 0 12.418 0 14a3 3 0 003 3h10a3 3 0 001.667-5.494.75.75 0 00-.834 1.246A1.5 1.5 0 1111.5 14c0-.642.225-1.347.623-2.136.397-.787.933-1.593 1.501-2.446l.011-.017c.554-.83 1.139-1.709 1.582-2.588.445-.885.783-1.836.783-2.818 0-1.672-1.346-3-3-3zm-10 1.5a1.5 1.5 0 00-1.5 1.5c0 .321.1.569.27.778.097.12.325.227.74.227h7.674A2.737 2.737 0 0110 3.995c0-.546.146-1.059.401-1.5H3zm10 0c.831 0 1.5.662 1.5 1.5 0 .646-.225 1.353-.623 2.143-.398.79-.933 1.595-1.501 2.448l-.017.026c-.552.828-1.134 1.702-1.575 2.576C10.338 12.072 10 13.021 10 14c0 .546.146 1.059.401 1.5H3A1.5 1.5 0 011.5 14c0-1.084.63-2.289 1.537-3.692.177-.274.366-.556.558-.845.632-.948 1.306-1.96 1.773-2.963h6.382a.75.75 0 00.417-1.373c-.444-.298-.667-.656-.667-1.132a1.5 1.5 0 011.5-1.5z"/></svg>
```

Comment:
@edokoa this file is using a height of `17` in the `height` and `viewBox` attributes.

Reply:
I saw Figma doing something strange with another icon and trying to push it as 15.99999999999999999 even if it was 16 in the interface. I'll re-upload  this one. Thanks for catching this.

Reply:
Sorry for not updating here, but this was corrected in https://github.com/primer/octicons/pull/715/commits/f48e121904a8f8e4c49dc37ef2e5d08ac63088fe